scripts/benchmarks/lib/bench.py

```diff
@@ -292,6 +362,19 @@ def default_path(self):
 
 
 class ZkappLimitsBenchmark(Benchmark):
+    """
+     ZkappLimit benchmark has it's own output which we need to handle separately. It is similar to:
+
+        Proofs updates=0  Signed/None updates=0  Pairs of Signed/None updates=1: Total account updates: 2 Cost: 10.080000
+        Proofs updates=0  Signed/None updates=0  Pairs of Signed/None updates=2: Total account updates: 4 Cost: 20.160000
+        Proofs updates=0  Signed/None updates=0  Pairs of Signed/None updates=3: Total account updates: 6 Cost: 30.240000
+        Proofs updates=0  Signed/None updates=0  Pairs of Signed/None updates=4: Total account updates: 8 Cost: 40.320000
+        Proofs updates=0  Signed/None updates=0  Pairs of Signed/None updates=5: Total account updates: 10 Cost: 50.400000
+        Proofs updates=0  Signed/None updates=0  Pairs of Signed/None updates=6: Total account updates: 12 Cost: 60.480000
+        Proofs updates=0  Signed/None updates=1  Pairs of Signed/None updates=0: Total account updates: 1 Cost: 9.140000
+        Proofs updates=0  Signed/None updates=1  Pairs of Signed/None updates=1: Total account updates: 3 Cost: 19.220000
+
+    """
```

Comment:
Just FYI, the cost here is computed using a formula whose inputs are the # of different account updates. So, for the same combination of account updates, the cost won't change unless the formula changes

Reply:
Hey, Yes i know that. I am open to debate regarding this test too. From one side it is not expected to change so frequently.  There is a threat that measurement would be static. From the other hand, someone developed zkapp_limit app so i guess it might be a reason to keep track of those numbers. If formula changes we will be alerted that cost changed. I don't know how important it is to know that, thought